cmd/restore-operator/README.md

```diff
@@ -0,0 +1,133 @@
+# etcd restore operator
+
+## Overview
+
+The etcd restore operator can restore a new etcd cluster on Kubernetes from an existing etcd backup snapshot.
+
+The overall workflow is:
+- Create the etcd-restore-operator
+- Create an `EtcdRestore` Custom Resource which triggers a restore request that specifies:
+  - the new cluster spec
+  - how to access the backup snapshot
+- The etcd-restore-operator will restore a new cluster from the backup
+- The etcd-operator takes over the management of the restored cluster
+
+The etcd-restore-operator only supports restoring from a backup snapshot file saved on AWS S3. See the [etcd-backup-operator README][backup-operator-README] on how to save a backup to S3.
+
+**Prerequisites:**
+- Setup RBAC rules and deploy the etcd-operator. See the [operator install guide][install-guide] and [RBAC guide][RBAC-guide].
+- Have an etcd backup snapshot file present in an S3 bucket. See the [etcd-backup-operator README][backup-operator-README] as one way to save a backup to S3.
+
+**Note:** This demo uses the `default` namespace.
```

Comment:
For consistency, do like https://github.com/coreos/etcd-operator/blob/master/cmd/backup-operator/README.md Note style